File: Day-01/PalindromeNumber.js
function reverseInteger(integer) {
  let rev = 0;

  while (integer > 0) {
    rev = rev * 10 + (integer % 10);
    integer = Math.floor(integer / 10);
  }

  return rev;
}

function checkPalindromeNumber(number) {
  const reverseNumber = reverseInteger(number);

  if (reverseNumber === number) {
    console.log("Number is Palindrome");
  } else {
    console.log("Number is not palindrome");
  }
}

checkPalindromeNumber(121);

File: Day-01/PalindromeNumber.py
def reverseInteger(number):
    rev = 0
    while(number > 0):
        rev = rev * 10 + number % 10
        number =int(number / 10)
    return rev

def isNumberPalindrome(number):
    reverseNumber = reverseInteger(number)
    if (reverseNumber == number):
        print("number is palindrome")
    else:
        print('Number is not palindrome')


isNumberPalindrome(1211)